### PR TITLE
Gutter image, clues, and golf cart fixes

### PR DESCRIPTION
**Pull request by catdad666**
Just a lot of little changes to The Gutter:
* Made Ron's cart sequence run smoother and read better for hatchet and multitool players. There's more explanation on why PC is destroying the cart.
* Talking to Toe without a game now includes some text about a shelf labeled "GAMES"
* Talking to Darryl about Toe now includes text referencing the games probably being in the Roach Market
* Added the Gutter image to the outside hub

### CODE REVIEW

File: src/s3a_the_gutter.tw2 [hunk 1]
```diff
@@ -367,10 +375,10 @@ Everyone at the bar was where I had left them. I got the feeling this bunch were
 
 He said nothing.
 
-(click: "nothing")[$dissolve[I waited. Still not a fucking peep.]]
+(click: "nothing")[$dissolve[I passed the time staring at a dusty, empty shelf behind Toe labeled "GAMES". Still not a fucking peep.]]
 
 (click: "peep")[(if: (text: $gameOfChance)'s length > 1)[$dissolve[I [[pulled out the $gameOfChance|The Gutter - Bar - Toe game start]]].]
-\(else:)[$dissolve[...I got [[bored|The Gutter - Bar hub]].]]]
+\(else:)[$dissolve[...I got [[bored.|The Gutter - Bar hub]]]]]
```
Comment (whymog) [inline src/s3a_the_gutter.tw2:381]: Most online writing styles (and the passages I've worked on) have terminating punctuation outside of a hyperlink. That seems like something we should go through and make consistent throughout the game before we finalize it.

**Comment by whymog** [comment]
Looks good to me! Please also add The Gutter's image to the `build/img` folder as well. If this branch isn't up-to-date with `master` and therefore doesn't have that directory, you can always do:

`git fetch`
`git checkout master`
`git pull`
`git checkout gutter-cart-clues-image`
`git rebase master`

Or we can just add it in another PR. That might be easier. Up to you!

**Review by whymog — APPROVED** [review]

File: src/s3a_the_gutter.tw2 (hunk 1 above)
Comment (catdad666) [reply src/s3a_the_gutter.tw2:381]: Yeah, I've been doing that same thing normally and I'll add it to the punch list. In this case, the period was hanging before the text loaded in. So this was an attempt to fix that.

**Comment by catdad666** [comment]
@whymog  Not sure what's going on with that. My master was up to date before I made this branch, but I had to make my own `build/img` folder. I added the gutter image to it, but it didn't come up in the PR.

Any attempt to check this branch or master says they're "tree clean" and that I don't have anything to commit.

I'm gonna go ahead and merge, but we can troubleshoot this tomorrow.

**Comment by whymog** [comment]
Did you run a `git fetch` and then `git checkout master` and `git status master` before merging? That'd tell you if you needed to rebase or not. Either way, your method should've worked fine too. Happy to explain in more detail if you're interested!

**Pull request merged**